libbeat/dashboards/importer.go

```diff
@@ -312,7 +304,7 @@ func (imp Importer) ImportKibanaDir(dir string) error {
 
 	check := []string{}
 	if !imp.cfg.OnlyDashboards {
-		check = append(check, "index-pattern")
+		imp.loader.ImportIndex()
```

Comment:
This breaks current behaviour in that it makes it impossible to load an index pattern from a file and requires the one from Beats.